tests/test_athaliana.py

```diff
@@ -0,0 +1,42 @@
+import unittest
+import io
+
+from stdpopsim import arabidopsis_thaliana
+
+
+class TestGenome(unittest.TestCase):
+    """
+    Tests for the arabidopsis_thaliana genome.
+    """
+
+    def test_basic_attributes(self):
+        genome = arabidopsis_thaliana.genome
+        self.assertEqual(genome.species, "arabidopsis_thaliana")
+        self.assertEqual(genome.default_genetic_map, "Salome2012")
+        self.assertEqual(len(genome.chromosomes), 5)
+
+    def test_str(self):
+        s = str(arabidopsis_thaliana.genome)
+        self.assertGreater(len(s), 0)
+        self.assertIsInstance(s, str)
```

Comment:
Is this doing anything (seeing as how `s` is the output of `str( )`)?

Reply:
looks like this unit test was coming straight from the earlier examples. not sure we want the `.assertIsInstance()`

Reply:
This is testing the ``__str__`` implementation on the Genome class. I'll tidy these all up in another pass after we've got this round of models in.